source/js/nav_jquery.js

```diff
@@ -1,8 +1,54 @@
 // Executes functions as soon as the page is fully loaded.
-$(document).ready(function() {
+$(document).ready(function () {
```

Comment:
This file is long and complicated. How can we make it easier to understand? (We can leave this concern for later)